docker/Dockerfile.release

```diff
@@ -9,6 +9,7 @@ ENV PROXY=${use_proxy:+'https_proxy=http://127.0.0.1:7890 http_proxy=http://127.
 
 COPY ./whls/* ${work_dir}/
 RUN cd ${work_dir} \
-    && env ${PROXY} pip install *.whl transformers==4.33.0 datasets pillow SentencePiece accelerate transformers_stream_generator tiktoken peft bitsandbytes scipy \
+    && env ${PROXY} pip install *.whl \
+    && env ${PROXY} pip install transformers==4.33.0 datasets pillow SentencePiece accelerate transformers_stream_generator tiktoken peft bitsandbytes scipy \
```

Comment:
Update transformers to higher version for llama3？

Reply:
Maybe we could update the version after @Seventeen17 optimize the model patches in a more robust way.